components/org.wso2.carbon.identity.workflow.impl/src/main/java/org/wso2/carbon/identity/workflow/impl/ExternalWorkflowTemplate.java

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright (c) 2023, WSO2 LLC. (http://www.wso2.com).
+ *
+ * WSO2 LLC. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.wso2.carbon.identity.workflow.impl;
+
+import org.wso2.carbon.identity.workflow.mgt.bean.metadata.InputData;
+import org.wso2.carbon.identity.workflow.mgt.exception.WorkflowException;
+import org.wso2.carbon.identity.workflow.mgt.exception.WorkflowRuntimeException;
+import org.wso2.carbon.identity.workflow.mgt.template.AbstractTemplate;
+
+import static org.wso2.carbon.identity.workflow.impl.constant.WorkflowConstant.APPROVAL_TEMPLATE_NAME;
+import static org.wso2.carbon.identity.workflow.impl.constant.WorkflowConstant.DESCRIPTION;
+import static org.wso2.carbon.identity.workflow.impl.constant.WorkflowConstant.TEMPLATE_ID;
+
+public class ExternalWorkflowTemplate  extends AbstractTemplate {
```

Comment:
Let's add class comment

Reply:
comment addressed : https://github.com/wso2-extensions/identity-workflow-impl-bps/pull/71/commits/9822ae47e6a0e5c4b9b37b0ab00bd2a3b6fdca64

Reply:
seems like this is not fixed